package.json

```diff
@@ -71,15 +66,5 @@
   },
   "engines": {
     "node": "0.12.x"
-  },
-  "jest": {
-    "scriptPreprocessor": "app/preprocessor.js",
-    "unmockedModulePathPatterns": [
-      "node_modules/react",
-      "node_modules/sundial",
-      "node_modules/moment",
-      "node_modules/lodash",
-      "node_modules/blip-mock-data"
-    ]
```

Comment:
Do we still have the Jest tests sitting somewhere? If so, is now the time to remove them? (I do want them duplicated eventually in our non-Jest setup, so if we still want them around for reference, I guess that's cool. They should be in __tests__ dirs in messages and, well, the other ones are probably on #217.)